device/device.yaml

```diff
@@ -17,21 +17,35 @@ components:
       x-include: 
       - '../common/common.yaml#/components/schemas/Named.Object'
       description: >-
-        A container for emulated interfaces and protocol devices.
+        A container for emulated interfaces, loopback interfaces and protocol configurations.
       type: object
-      required: [container_name, ethernet]
       properties:
-        container_name: 
+        ethernets:
           description: >-
-            The unique name of a Port or Lag object that will contain the 
-            emulated interfaces and/or protocol devices.
-          type: string
-          x-constraint:
-          - '/components/schemas/Port/properties/name'
-          - '/components/schemas/Lag/properties/name'
-        ethernet:
+            The emulated Ethernet settings.
+          type: array
+          items:
+            $ref: './ethernet.yaml#/components/schemas/Device.Ethernet'
+        ipv4_loopbacks:
           description: >-
-            The ethernet stack.
-          $ref: './ethernet.yaml#/components/schemas/Device.Ethernet'
-    
-  
+            IPv4 Loopback interface settings attached to an Ethernet interface.
```

Comment:
Should be rephrased to mention:

> IPv4 loopback address configuration for a network interface defined in any device emulated by a port which emulates current device.

Reply:
IPv4 Loopback interface can be attached to an Ethernet in the same device  or to an Ethernet in another device.

Reply:
*that* is missing (same for IPv6).

IPv4 Loopback interface can … -> IPv4 Loopback interface *that* can …